src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx

```diff
@@ -72,27 +83,35 @@ export class NodeContextMenu extends React.PureComponent<NodeContextMenuProps, N
   }
 
   render() {
-    const version = this.props.version !== '' ? `:${this.props.version}` : '';
-    const detailsPageUrl = this.makeDetailsPageUrl();
-    const { nodeType, app } = this.state;
+    // Disable context menu if we are dealing with a unknown node
```

Comment:
We should also disable content menu for isInaccessible nodes.  These nodes are not actually accessible to the user, they are rendered only to show a connection and are typically in a restricted namespace or may be service entries.

Reply:
Sounds good

Reply:
Done